python/task/download.py

```diff
@@ -53,6 +55,8 @@ def run(self, task):
     # Download from url to file.
     if ratelimit > 0: log.info("Start download of " + url)
     conn = urllib2.urlopen(url)
+    srvLastModified = time.mktime(time.strptime(conn.headers['last-modified'],
```

Comment:
The python convention is to use lowercase with underscores for variable names.

Reply:
Thanks. Fixed.